doc/2/api/controllers/document/update-by-query/index.md

```diff
@@ -0,0 +1,109 @@
+---
+code: true
+type: page
+title: updateByQuery
+---
+
+# updateByQuery
+
+Updates documents matching the provided search query. 
+
+Documents updated that way trigger real-time notifications.
+
+## Limitations
+
+The request fails if the number of documents returned by the search query exceeds the `documentsWriteCount` server configuration (see the [Configuring Kuzzle](/core/2/guides/essentials/configuration) guide).
+
+To update a greater number of documents, either change the server configuration, or split the search query.
+
+---
+
+## Query Syntax
+
+### HTTP
+
+```http
+URL: http://kuzzle:7512/<index>/<collection>/_queryUpdate
+Method: PUT
+Body:
+```
+
+```js
+{
+  "query": {
+    // ...
+  }
+}
+```
+
+### Other protocols
+
+```js
+{
+  "index": "<index>",
+  "collection": "<collection>",
+  "controller": "document",
+  "action": "updateByQuery",
+  "body": {
+    "query": {
+      // query to match documents
+    },
+    "changes": {
+      // documents changes
+    }
+  }
+}
+```
+
+---
+
+## Arguments
+
+- `collection`: collection name
+- `index`: index name
+
+---
+
+### Optional:
+
+- `source`: if set to `true` Kuzzle will return the updated documents body in the response.
+
+## Body properties
+
+- `query`: documents matching this search query will be updated. Uses the [ElasticSearch Query DSL](https://www.elastic.co/guide/en/elasticsearch/reference/7.4/query-dsl.html) syntax.
+
+---
+
+## Response
```

Comment:
I think it will be better to names the 2 arrays `successes` and `errors` like for the `m*` methods of the API.  

See https://docs.kuzzle.io/core/2/api/controllers/document/m-update/


Also, the `total` property is useless since you just have to compute the `successes` array length.